doc/getting_started/up2.rst

```diff
@@ -15,7 +16,7 @@ N4200* <https://ark.intel.com/products/95592/Intel-Pentium-Processor-N4200-2M-Ca
 SoCs. Both have been confirmed to work with ACRN.
 
 Connecting to the serial port
------------------------------
+=============================
 
 The UP2 board has two serial ports: one via the M10 header and one via
 the I/O pins in the 40-pin HAT connector. Please refer to the `*UP2
```

Comment:
We avoid using link text such as "here" or "this one", so rewrite the sentence below to be:

    To connect to the UP2 board, use a `USB TTL serial cable
    <http://www.ftdichip.com/Products/USBTTLSerial.htm>`__.